CONTRIBUTING.md

```diff
@@ -64,6 +65,32 @@ issue yourself. This gives the team visibility of the work you plan to do.
 The other advantage to raising the issue at the outset is that our process requires an
 issue to be associated with every PR (see [patch format](#patch-format)).
 
+### Before submitting a PR
+
+Kata Containers utilizes a number of [CI systems](https://en.wikipedia.org/wiki/Continuous_integration)
+to automatically check every PR. As well as performing execution tests locally on
+your PR before submission, you are also strongly encouraged to run the suite of
+[static checks](https://github.com/kata-containers/tests/blob/master/.ci/static-checks.sh)
+the CI will execute. The easiest way to execute these in most of the repositories is
+to invoke the `Makefile` `check` rule, whilst enabling developer mode:
+
+```sh
+$ export KATA_DEV_MODE=true
+$ make check
+```
+
+Execution of this should complete with no errors. If errors are reported, fix them
+before submitting your PR.
+
+> **Note:** To ensure you replicate the static checks performed by the CI system, it is
+recommended that you:
+> * Ensure you have a 'clean' source tree, as the checks will check all files present, and
+may fail if you have out of date or extra files in your tree.
+> * Ensure you have an up-to date (or have not installed, as the static check scripts will
```

Comment:
Lines 89-92 suggested rewrite (hopefully I did not change meaning): 
* Ensure that [gometalinter](https://github.com/alecthomas/gometalinter) is current or has not been installed due to the fact that the static check scripts will install. Changes in either the linters used or the Kata Containers code base can produce spurious errors that do not fail inside the CI systems.